daemons/dss-checkout-sfn/app.py

```diff
@@ -0,0 +1,96 @@
+import os
+import sys
+
+import domovoi
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), 'domovoilib'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+import dss
+from dss.util.state_machine.checkout_states import state_machine_def
+from dss.util.email import send_checkout_success_email, send_checkout_failure_email
+from dss.util.checkout import (parallel_copy, get_dst_bundle_prefix, get_manifest_files,
+                               validate_file_dst, validate)
+
+app = domovoi.Domovoi()
+dss.Config.set_config(dss.BucketConfig.NORMAL)
+
+replica = "aws"
```

Comment:
My hunch is that replica should be a parameter to the step function, i.e., it should come in on the event object.